Sources/Latte+ProofOfPhoneNumberVerification.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+public struct ProofOfPhoneNumberVerificationResponse: Decodable {
+    let proofOfPhoneNumberVerification: String
+}
+
+public extension Latte {
+    func getProofOfPhoneNumberVerification(
+        handler: @escaping (Result<String, Error>) -> Void
+    ) {
+        var proofOfNumberVerificationEndpoint = self.middlewareEndpoint + "/proof_of_phone_number_verification"
+        var request = URLRequest(url: URL(string: proofOfNumberVerificationEndpoint)!)
+        request.httpMethod = "POST"
+
+        let jsonEncoder = JSONEncoder()
+        let body = try! jsonEncoder.encode([
+            "access_token": authgear.accessToken
```

Comment:
Similar to https://github.com/authgear/authgear-sdk-android/pull/203/files#r1971234394, refresh the access token first.